### ISSUE TITLE
Error "_Link.Link is not a constructor" when submit form

### ISSUE THREAD

**Issue by Err0r404** [swup/forms-plugin#14]
**Describe your issue**:
Using symfony/ux-swup from https://github.com/symfony/ux-swup that's also uses swup/forms-plugin and can't get my form to works with swup : error pop in console and page reload like regular form.

**Swup config or any additional relevant code used**:

- Default configuration from symfony/ux-swup
- Demo available here : https://github.com/Err0r404/swup-demo

Generated form
```
<form name="contact" method="post" data-swup-form="data-swup-form">
            <div class="form-group"><label for="contact_name" class="required">Name</label><input type="text" id="contact_name" name="contact[name]" required="required" class="form-control" value="John Doe"></div>
            <div class="text-right">
                <button role="button" type="submit" class="btn btn-lg btn-primary">Send</button>
            </div>
            <input type="hidden" id="contact__token" name="contact[_token]" value="OnuzpBXVIllIVz9LFel1toXlt1ORvZo9E7a9kBNDsik">
</form>
```

Error in console
```
Uncaught TypeError: _Link.Link is not a constructor
    at FormPlugin.onFormSubmit (index.js:78)
    at HTMLDocument.<anonymous> (delegate.js:73)
```

**Before creating this issue, did you think of...**:
- [x] Have you checked closed issues for similar/related problems.
- [x] Have you provided all helpful information available? 
- [x] Have you considered creating a demo so we can help you better?

**Comment by daun** [comment]
I'm pretty sure you need an `action` on that form so that swup knows where to send the data.

**Comment by Err0r404** [comment]
Thanks for the help!

Actually without the `action` attribute the form is submitted to the same page.

But following your advice I tryed to set it but the same error is still here.

Tryed different options without success:
- relative url (action="/contact")
- absolute url (action="https://127.0.0.1:8000/contact")
- different relative url (action="/thanks")
- different absolute url (action="https://127.0.0.1:8000/thanks")

Error is linked to `var link = new _Link.Link(actionAttribute);` in `function onFormSubmit(event)`

**Comment by daun** [comment]
Oh, right. Now I remember where I saw that error message.

There's a fix in PR #13.

**Comment by Err0r404** [comment]
@daun OK Thanks for your precious help, I can confirm that PR #13 fix form submission!

Can't wait to have it merged 👍

**Comment by ahmed-bhs** [comment]
@Err0r404 could you please specify how to apply the patch exactly because it doesn't work for me and I'm still facing the same issue !

**Comment by Err0r404** [comment]
I have manually included `src` folder in my project, manually applied the correction from PR, compiled to get new scripts in `dist` and then used those new compiled scripts

**Comment by gmrchk** [comment]
Apologies for the inconvenience. Must have forgotten to update the plugin during some refactoring. 
Fixed in v1.1.1 by @fracsi.
